feed.xml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Works very smoothly by default. I've added the auto-discovery helper but I'm not sure if it's working for me. Can someone else confirm?